thavalon-webapp/src/utils/AccountManager.tsx

```diff
@@ -234,43 +244,41 @@ export class AccountManager {
             "message": ""
         }
 
-        return await fetch("/api/auth/login", {
+        let response: Response = await fetch("/api/auth/login", {
             method: "POST",
             body: JSON.stringify(logInInfo),
             headers: {
                 "Content-Type": "application/json"
```

Comment:
These are out of order since I'm scrolling up. Ditto on quotes. See below for what is dittoed.